File: mwahanje.py
class Roommate:
    def __init__(self, name, age, home, school, family, hobbies):
        self.name = name
        self.age = age
        self.home = home
        self.school = school
        self.family = family
        self.hobbies = hobbies

    def display_info(self):
        print("Roommate Information:")
        print("---------------------")
        print(f"Name: {self.name}")
        print(f"Age: {self.age} years old")
        print(f"Home: {self.home}")
        print(f"School: {self.school}")
        print(f"Family: {self.family}")
        print("Hobbies: " + ", ".join(self.hobbies))


# Example usage
rm = Roommate(
    name="John Doe",
    age=22,
    home="Nairobi, Kenya",
    school="Juja University",
    family="Lives with parents and two siblings",
    hobbies=["Football", "Reading", "Coding", "Music"]
)

rm.display_info()

File: mwahanje.cpp
#include <iostream>
using namespace std;

int main() {
    string name = "my roomate's name is #include <iostream>

class Roommate {
public:
    string name;
    int age;
    string home;
    string school;
    string family;
    vector<string> hobbies;

 {
        cout << "Roommate Information:" << endl;
        cout << "---------------------" << endl;
        cout << "Name: " << name << endl;
        cout << "Age: " << age << " years old" << endl;
        cout << "Home: " << home << endl;
        cout << "School: " << school << endl;
        cout << "Family: " << family << endl;
        cout << "Hobbies: ";
        for (size_t i = 0; i < hobbies.size(); i++) {
            cout << hobbies[i];
            if (i != hobbies.size() - 1) cout << ", ";
        }
        cout << endl;
    }

int main() {
    Roommate rm;

    rm.name = "John Doe";
    rm.age = 22;
    rm.home = "Nairobi, Kenya";
    rm.school = "Juja University";
    rm.family = "Lives with parents and two siblings";
    rm.hobbies = {"Football", "Reading", "Coding", "Music"};

    return 0;
}
